src/actions.js

```diff
@@ -257,14 +254,24 @@ export function fetchRegionDistricts(parent) {
   return graphql(payload, `LOCATION_REGION_DISTRICTS`);
 }
 
+export function fetchDataFromDistrict(districtUuids) {
+  const filters = [];
+  if (districtUuids) {
+    filters.push(`parent_Uuid_In: ["${districtUuids.join('", "')}"]`);
+  }
+  const payload = formatPageQuery("locations", filters, [
+    "id, uuid, code, name, parent { id, uuid, name, code }, children { edges {node {id, uuid, code, name, parent { id, uuid, code, name}}}}",
+  ]);
+  return graphql(payload, `LOCATION_DISTRICT_DATA`);
+}
 
 export function fetchObligatoryUserFields() {
-  let payload = "query userObligatoryFields {userObligatoryFields}"
+  let payload = "query userObligatoryFields {userObligatoryFields}";
   return graphql(payload, `OBLIGTORY_USER_FIELDS`);
 }
 
 export function fetchObligatoryEnrolmentOfficerFields() {
-  let payload = "query userObligatoryFields {eoObligatoryFields}"
+  let payload = "query userObligatoryFields {eoObligatoryFields}";
```

Comment:
I wonder if we should use 'const' instead of 'let'. This seems not to be changed when it comes to the value of this variable.

Reply:
Changed.